dlc_developer_config.toml

```diff
@@ -34,11 +34,11 @@ deep_canary_mode = false
 [build]
```

Comment:
The images being currently built on the PR are PT2.1. 

Please change this https://github.com/aws/deep-learning-containers/pull/3602/files#diff-8830611925a0a24696630b322ccaf048c62308faaabdac2d917be57d48c7ff4bR102 to `dlc-pr-pytorch-training =  "pytorch/training/buildspec-2-0-sm.yml"`

Reply:
corrected